README.md

```diff
@@ -1,44 +1,97 @@
+
+## Usage
+
+Before getting started with ml5.js, review our [Code of Conduct](https://github.com/ml5js/Code-of-Conduct). There are several ways you can use the ml5.js library:
+
+<p id="latest-version">
+
+- You can use the latest version (0.12.2) by adding it to the head section of your HTML document:
+
+**v0.12.2**
+
+    <script src="https://unpkg.com/ml5@0.12.2/dist/ml5.min.js" type="text/javascript"></script>
+
+</p data-id="latest-version">
+
+- If you need to use an earlier version for any reason, you can change the version number. The [previous versions of ml5 can be found here](https://www.npmjs.com/package/ml5?activeTab=versions). You can use those previous versions by replacing `<version>` with the ml5 version of interest:
+
+```html
+<script
+  src="https://unpkg.com/ml5@<version>/dist/ml5.min.js"
+  type="text/javascript"
+></script>
 ```
 
-You should see something similar to this in the terminal:
+For example:
 
+```html
+<script
+  src="https://unpkg.com/ml5@0.6.1/dist/ml5.min.js"
+  type="text/javascript"
+></script>
 ```
-[webpack-dev-server] Project is running at:
-[webpack-dev-server] Loopback: http://localhost:8080/
-...
-...
-webpack 5.76.1 compiled successfully in 8360 ms
+
+- You can also reference "latest", but we do not recommend this as your code may break as we update ml5.
+
+```html
+<script
+  src="https://unpkg.com/ml5@latest/dist/ml5.min.js"
+  type="text/javascript"
+></script>
 ```
 
-A local server have been started and hosts a built version of the ml5 library at http://localhost:8080/dist/ml5.js. While the server is running, Webpack will automatically rebuild the library if you change and save any file in the `/src` folder.
+Welcome to the next-generation repo for the ml5.js project! Due to the complexities inherent in updating all the dependencies in [the original repo](https://github.com/ml5js/ml5-library), we're exploring rebuilding from the ground up. This fresh repository allows us to migrate over classes and functions one at a time. This is very much a work in progress and is currently in its early, experimental stage. We appreciate your patience as we work towards refining this next evolution of the ml5.js project. Stay tuned!
+
+## Resources
+
+- [Getting Started](https://learn.ml5js.org/)
+- [API Reference](https://learn.ml5js.org/#/reference/index)
+- [Examples](https://github.com/ml5js/ml5-library/tree/main/examples)
+- [Community](https://ml5js.org/community)
+- [FAQ](https://learn.ml5js.org/#/faq)
+
+## Standalone Examples
+
+You can find a collection of standalone examples in this repository within the `examples/` directory. You can also test working hosted of the examples online on the [ml5.js examples index website](https://examples.ml5js.org/).
+
+These examples are meant to serve as an introduction to the library and machine learning concepts.
```

Comment:
Are we still maintaining this?

Reply:
I think we are deprecating this, is that correct @MOQN?

Reply:
@shiffman Sorry for my slow response! Yes, it is correct. We will discuss and develop a different approach to share the example codes.